interpreter/namerd/src/main/scala/io/buoyant/namerd/iface/NamerdInterpreterInitializer.scala

```diff
@@ -110,3 +133,18 @@ case class NamerdInterpreterConfig(
 object NamerdInterpreterConfig {
   def kind = classOf[NamerdInterpreterConfig].getCanonicalName
 }
+
+object FailureThresholdConfig {
+
+  val DefaultMinPeriod = 5.seconds
+  val DefaultThreshold = 2.0
+  val DefaultWindowSize = 100
+  val DefaultCloseTimeout = 4.seconds
```

Comment:
After discussing with @olix0r, I think we should make this default to Duration.Top